crates/polars-core/src/datatypes/dtype.rs

```diff
@@ -542,11 +543,15 @@ impl DataType {
             Float32 => Ok(ArrowDataType::Float32),
             Float64 => Ok(ArrowDataType::Float64),
             #[cfg(feature = "dtype-decimal")]
-            // note: what else can we do here other than setting precision to 38?..
-            Decimal(precision, scale) => Ok(ArrowDataType::Decimal(
-                (*precision).unwrap_or(38),
-                scale.unwrap_or(0), // and what else can we do here?
-            )),
+            Decimal(precision, scale) => {
```

Comment:
We should probably also check that `scale <= precision`.

Reply:
I don't believe scale is limited: https://arrow.apache.org/docs/python/generated/pyarrow.decimal128.html